src/main/scala/scalapb/json4s/JsonFormat.scala

```diff
@@ -157,9 +157,9 @@ class Printer(
     }
   }
 
-  private def serializeNonMessageField(fd: FieldDescriptor, name: String, value: PValue, b: FieldBuilder) = {
+  private def serializeNonMessageField(fd: FieldDescriptor, name: String, value: PValue, b: FieldBuilder, oneofs: Set[Int]) = {
```

Comment:
Instead of the additional parameter, wouldn't it be possible to do `fd.containingOneOf.isDefined` ?

Reply:
Changed to `fd.containingOneOf` somehow I managed to miss it.